src/main/java/dumshenko/daniil/Circle.java

```diff
@@ -0,0 +1,23 @@
+package dumshenko.daniil;
+
+public class Circle extends Shape {
+
+    private double radius;
+
+    public Circle(FigureType typeOfFigure, double radius) {
+        super(typeOfFigure);
+        this.radius = radius;
+    }
+
+    @Override
+    public void displayInfo() {
+        System.out.println("I`m Circle ");
+        System.out.println("My radius: " + radius);
+    }
+
+    @Override
+    public double calculateArea() {
+        figureArea = (3.14159 * (radius * radius)); //Формула для обчислення площі круга
+        return figureArea;
```

Comment:
nit: Don't forget that we have a `Math` helper class in java. The assignment to variable is unnecessary here, we can directly return the area.
```suggestion
        return Math.PI * Math.pow(radius, 2.0); //Формула для обчислення площі круга
```